templates/bot/ts/notification/src/adaptiveCard.ts

```diff
@@ -5,13 +5,10 @@ export type AdaptiveCard = any;
 
 /**
  * Build adaptive card payload with card template and data.
- * @param getCardData Function to prepare your card data.
+ * @param cardData The adaptive card data.
  * @param cardTemplate The adaptive card template.
  * @returns An adaptive card object.
  */
-export function buildAdaptiveCard<TData>(
-  getCardData: () => TData,
-  cardTemplate: any
-): Partial<Activity> {
-  return AdaptiveCards.declare<TData>(cardTemplate).render(getCardData());
+export function buildAdaptiveCard<TData>(cardData: TData, cardTemplate: any): Partial<Activity> {
```

Comment:
You can remove `adaptiveCard.ts` since the `MessageBuilder` utility already included in the SDK.

Reply:
Took a look at `MessageBuilder` and it seems to return an `Activity`, but here `buildAdaptiveCard` prefers a card json. (To keep user code consistency with Webhook).

I'll try `MessageBuilder` in later PR for Webhook as well.

Reply:
I see. I should also include the `AdaptiveCard` builder APIs in `MessageBuilder` to build adaptive card json without wrapping in an `attachment` :).